framework/src/systems/NonlinearEigenSystem.C

```diff
@@ -155,6 +169,11 @@ NonlinearEigenSystem::initialSetup()
     // IntegratedBCs
     addPrecondTagToMooseObjects(_integrated_bcs);
   }
+
+  // Mark a variable an eigen variable if it operates on eigen kernels
+  markEigenVariables(_dg_kernels);
+  markEigenVariables(_kernels);
+  markEigenVariables(_scalar_kernels);
```

Comment:
This looks tricky. IIRC we cannot do eigen with (integrated) bcs? In which case, I assume it wouldn't make sense either to do them with interface kernels? I think this probably doesn't make sense thinking about the strong form of the eigenvalue problem

Reply:
We support `integrated bcs` now.` interface kernels` should make sense, but there is no use case yet.